blazor/message/variants.md

```diff
@@ -69,4 +69,8 @@ The following example demonstrates the default message with different variant ty
 {% endhighlight %}
 {% endtabs %}
 
-![Message Variants](./images/message-variants.png)
+![Filled Variant](./images/variants-filled.png)
+
+![Outlined Variant](./images/variants-outlined.png)
+
```

Comment:
take the image in bootstrap 5 theme

Reply:
Replaced all the images with bootstrap5 theme